Combine two table of contents into one

rontu85 opened the pull request:
I've merged the two table of contents into one. They are both shown differently if on mobile or desktop. As mention in Issue [#96](https://github.com/tightenco/onramp/issues/96) aria-hidden could be removed since we just have one table of contents left.

mattstauffer commented:
I'm OK with considering this idea, but as we talked about on the stream, this is no longer a generic `<Toggle>` component, it's now very specific to this use case.

I actually think I'm OK with that, but we would want to rename it something like `toggleWhenMobile` or something like that. Also, change the internal variables to be not just about table of contents.

I also still think we would want the aria role; when it's hidden on mobile, it should still get that aria role, right?

rontu85 commented:
I've renamed it, as you mentioned. Otherwise I try to come up with another solution that is more generic.

I'm no expert regarding ARIA syntax, but if we let aria-hidden in, there would be no table of contents for aria screen readers. Maybe someone of the others knows more about this topic.

mattstauffer commented:
Oh, right, duh, I was mixing up the ideas. Thanks!

mattstauffer commented on resources/js/components/ToggleWhenMobile.vue:
```diff
@@ -26,6 +26,12 @@
             buttonLabel() {
                 return this.isOpen ? 'Hide' : 'Show';
             },
+            slotVisible() {
```
I think I'd rather evaluate this by adding and removing Tailwind classes instead of evaluating it in JavaScript.

What I imagined was:

```
md:block (so it always shows on md)
hidden (class enabled or disabled based on the isOpen)
```

rontu85 replied:
Could you please take a look? It should be resolved now.

rontu85 commented:
Hello @mattstauffer, I've implemented the changes as you wished. Could you please take a look, if this is corret for you?

I've rebased my branch with your master branch and force pushed it. As a result all comments are not in sync with the commits anymore. This was not what I've had in mind but I wanted to test out rebasing with force push. For future PRs do you prefer to resolve branch conflicts yourself or should I do it upfront?

mattstauffer commented:
Thanks!
